libevmasm/RuleList.h

```diff
@@ -47,6 +47,11 @@ template <class S> S modWorkaround(S const& _a, S const& _b)
 	return (S)(bigint(_a) % bigint(_b));
 }
 
+inline u256 shlWorkaround(u256 const& _x, unsigned _amount)
```

Comment:
There is a helper for this in libdevcore, IIRC.

Reply:
See https://github.com/ethereum/solidity/blob/develop/libdevcore/CommonData.h#L191